package.json

```diff
@@ -92,13 +94,16 @@
     "jest-cli": "^27.3.1",
     "magic-string": "^0.25.7",
     "nollup": "^0.18.6",
-    "react": "^17.0.2",
+    "react": "^18.0.0",
     "react-16": "npm:react@^16.14.0",
-    "react-dom": "npm:@hot-loader/react-dom@^17.0.1",
+    "react-17": "npm:react@^17.0.2",
+    "react-dom": "^18.0.0",
```

Comment:
There's no `npm:@hot-loader/react-dom@^18.0.0` yet, so I used the normal `react-dom` package. Of course it would be great to use the `@hot-loader/react-dom` one for all supported React versions.